test/e2e/driver/azuredisk_driver.go

```diff
@@ -123,6 +123,6 @@ func (d *azureDiskDriver) GetPersistentVolume(volumeID string, fsType string, si
 
 func GetParameters() map[string]string {
 	return map[string]string{
-		"skuName": "StandardSSD_LRS",
+		"skuName": "Standard_LRS",
```

Comment:
we still want to test other `skuName` for public cloud. Should add a check here whether it's AzureStack.